src/app/shared/navigation/navigation.component.ts

```diff
@@ -85,37 +85,29 @@ export class NavigationComponent {
   public constructor(
     private readonly navigationListService: NavigationListService,
     private readonly preferenceService: PreferenceService,
-    private readonly navListComponentService: NavigationListComponentService,
     private readonly activatedRoute: ActivatedRoute,
-    private readonly timeRangeService: TimeRangeService,
-    private readonly pageTimeRangeService: PageTimeRangePreferenceService,
-    private readonly loggerService: LoggerService
+    private readonly timeRangeService: TimeRangeService
   ) {
-    const navItems = this.navItemDefinitions.map(definition =>
+    this.navItems = this.navItemDefinitions.map(definition =>
       this.navigationListService.decorateNavItem(definition, this.activatedRoute)
     );
-    // Decorate the nav items with the corresponding time ranges, depending on the FF state.
-    // The time ranges in nav items are streams that get the most recent value from page time range preference service
-    this.navItems$ = this.navListComponentService.resolveNavItemConfigTimeRanges(navItems);
 
     this.isCollapsed$ = this.preferenceService.get(NavigationComponent.COLLAPSED_PREFERENCE, false);
+    this.updateDefaultTimeRangeIfUnset();
```

Comment:
Are we sure this isn't too early? Looks like before it was only being initialized once the active item was set  which was due to the requirement to check what the item was. We don't care what the item is any more, but we do need to make sure this code isn't called before nav service's regular init has completed and applied any deep linked time range.

Reply:
@aaron-steinfeld I think we don't need to call this method from here. The TR is already getting initialized. I should probably move the default time range code to the TR service. It is possible that we are exposing a public method `updateDefaultTimeRangeIfUnset()` so that we can set different default tr from outside. 

```
  public constructor(
    private readonly navigationService: NavigationService,
    private readonly timeDurationService: TimeDurationService
  ) {
    this.initializeTimeRange();
    this.navigationService.registerGlobalQueryParamKey(TimeRangeService.TIME_RANGE_QUERY_PARAM);
  }
 ```

Reply:
Please also check the TR service changes in the latest commit.

Reply:
Do we even need this method and call anymore since there is only one way to set a timerange? The navigation component should not care about what time range is set?

Reply:
+1 - please move the default time range assignment into TR service. Otherwise the changes to TR service look good.